Mapper/BaseGroupedMapper.php

```diff
@@ -70,11 +70,17 @@ public function with($name, array $options = array())
             'collapsed' => false,
             'class' => false,
             'description' => false,
+            'label' => $this->admin->getLabelTranslatorStrategy()->getLabel($name, $this->getName(), 'group'),
             'translation_domain' => null,
             'name' => $name,
             'box_class' => 'box box-primary',
         );
 
+        // BC layer for manual label
+        if (strpos($name, '.')) {
+            $defaultOptions['label'] = $name;
+        }
```

Comment:
Ok, however it will only deport the problem in the `AbstractAdmin` class, which initialize the `FormMapper` and the `ShowMapper` : https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Admin/AbstractAdmin.php#L1265